### PR TITLE
Modify fitting parameter

### PR DESCRIPTION
**Pull request by aogaki**
I am seriously sorry for not checking correctly before pull request.
The previous commit has a problem with using NOT user-defined parameter names.  
Now the wrong logic is corrected (change sequence).

### CODE REVIEW

File: scripts/JSRootCore.js [hunk 1]
```diff
@@ -1949,15 +1949,15 @@
             return this._func(x, y);
          }
          m.GetParName = function(n) {
-            if (this.fParams && this.fParams.fParNames) return this.fParams.fParNames[n];
-            if (this.fFormula && this.fFormula.fParams) return this.fFormula.fParams[n].first;
+	    if (this.fFormula && this.fFormula.fParams) return this.fFormula.fParams[n].first;
             if (this.fNames && this.fNames[n]) return this.fNames[n];
-            return "p"+n;
+            if (this.fParams.fParNames[n]!==undefined) return this.fParams.fParNames[n];
```
Comment (linev) [inline scripts/JSRootCore.js]: You should check that this.fParams exists - see my fixes to your previous PR.
``` 
 if (this.fParams && this.fParams.fParNames) return this.fParams.fParNames[n];
```
Please fix it before I can merge

File: scripts/JSRootCore.js [hunk 2]
```diff
@@ -1949,15 +1949,15 @@
...
+	    return "p"+n;
          }
          m.GetParValue = function(n) {
-            if (this.fParams && this.fParams.fParameters) return this.fParams.fParameters[n];
-            if (this.fFormula && this.fFormula.fClingParameters) return this.fFormula.fClingParameters[n];
-            if (this.fParams) return this.fParams[n];
+	    if (this.fFormula && this.fFormula.fClingParameters) return this.fFormula.fClingParameters[n];
+            if (this.fParams.fParameters[n]!==undefined) return this.fParams.fParameters[n];
```
Comment (linev) [inline scripts/JSRootCore.js]: Same here 
``` 
if (this.fParams && this.fParams.fParameters && this.fParams.fParameters[n] != undefined) return this.fParams.fParameters[n];
```

**Comment by linev** [comment]
Also please rebase your code - I changed it after your previous PR.

**Comment by aogaki** [comment]
Once.  I close this request and will make a new one.  After corrections

**Pull request closed without merging**